extras/ams_wrapper/src/api/types.py

```diff
@@ -86,7 +87,9 @@ def as_dict(self):
         result_dict = {
             "type": self.type_name,
             "subtype": self.subtype_name,
-            "classification": self.attributes[0].as_dict()
+            "classification": {
+            	"tag": attribute.as_dict() for attribute in self.attributes
```

Comment:
Since there can be only one tag for classification why use list of attributes? 
You could have separate class `Tag `with required fields and go with something like `"tag": self.tag.as_dict()`
It would increase readability and separate tag from attribute (in case you needed to add attributes to the classification output in the future).
Despite that, there will be only one tag element in classification, so using list of attributes can be misleading as only the last element of attributes list will be present in this dict.

Reply:
Committed the new changes. Kindly review